### DISCUSSION TITLE
Welcome to python-poloniex Discussions!

### DISCUSSION THREAD

**Discussion by s4w3d0ff** [s4w3d0ff/python-poloniex#211] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sahelanthropussy** [comment]
Hi @s4w3d0ff, how would I get my order id when I make an order using:
`order = polo.buy("USDT_BTC", price, amount)
        return order`
??
The underlying POST  returns the orderNumber but I don't know how to get its value.
Cheers
